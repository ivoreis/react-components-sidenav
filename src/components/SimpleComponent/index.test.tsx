import React from 'react';
import { render } from 'react-testing-library';
import { SimpleComponent } from '.';

test('it works', () => {
  const { container } = render(<SimpleComponent title={'My title'} />);
  expect(container.firstChild).toMatchSnapshot();
})