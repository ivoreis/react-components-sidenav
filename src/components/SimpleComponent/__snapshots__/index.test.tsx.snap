// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  background: #353644;
  color: #1edc8c;
  font-weight: bold;
  font-size: 24px;
  padding: 25px;
}

<div
  class="c0"
>
  My title
</div>
`;
